static/app/components/stream/groupChart.tsx

```diff
@@ -19,70 +19,49 @@ function asChartPoint(point: [number, number]): {name: number | string; value: n
 const EMPTY_STATS: ReadonlyArray<TimeseriesValue> = [];
 
 type Props = {
-  data: Group;
-  statsPeriod: string;
+  stats: ReadonlyArray<TimeseriesValue>;
   height?: number;
+  secondaryStats?: ReadonlyArray<TimeseriesValue>;
   showMarkLine?: boolean;
   showSecondaryPoints?: boolean;
 };
 
 function GroupChart({
-  data,
-  statsPeriod,
-  showSecondaryPoints = false,
+  stats,
   height = 24,
+  secondaryStats = EMPTY_STATS,
+  showSecondaryPoints = false,
   showMarkLine = false,
 }: Props) {
-  const stats: ReadonlyArray<TimeseriesValue> = statsPeriod
-    ? data.filtered
-      ? data.filtered.stats?.[statsPeriod]
-      : data.stats?.[statsPeriod]
-    : EMPTY_STATS;
-
-  const secondaryStats: TimeseriesValue[] | null =
-    statsPeriod && data.filtered ? data.stats[statsPeriod] : null;
-
-  const [series, colors, emphasisColors]: [
-    Series[],
-    [string] | undefined,
-    [string] | undefined,
-  ] = useMemo(() => {
+  const graphOptions = useMemo<{
+    colors: [string] | undefined;
+    emphasisColors: [string] | undefined;
+    series: Series[];
+  }>(() => {
     if (!stats || !stats.length) {
-      return [[], undefined, undefined];
+      return {colors: undefined, emphasisColors: undefined, series: []};
     }
 
-    let max = 0;
-    const marklinePoints: number[] = [];
-
-    for (let i = 0; i < stats.length; i++) {
-      const point = stats[i];
-      if (point[1] > max) {
-        max = point[1];
-      }
-      marklinePoints.push(point[1]);
-    }
```

Comment:
Not sure what the purpose of this marklinePoints array was, but it was not being used anywhere.